src/etl/arbo/steps/jitter-pin-lat-lng-step.ts

```diff
@@ -34,7 +34,7 @@ export const jitterPinLatLngStep = (
   const { allEstimates, allSources } = input;
   const maximumPinJitterMagnitude = 0.1;
 
-  console.log(`Running step: jitterPinLatLngStep. Remaining estimates: ${input.allEstimates.length}`);
+  console.log(`Running step: jitterPinLatLngStep. Remaining estimates: ${input.allEstimates.length}. Maximum pin jitter mangnitude: ${maximumPinJitterMagnitude}`);
```

Comment:
curious as to why?

Reply:
Hmmmm I think this was to make sure the thing is actually set to the correct value when I'm running it. I don't know, something might have burned me a month ago hahaha